ci/github/unit-testing/director_v2.bash

```diff
@@ -10,7 +10,7 @@ install() {
 }
 
 test() {
-    pytest --cov=simcore_service_director_v2 --durations=10 --cov-append \
+    pytest --numprocesses=auto --cov=simcore_service_director_v2 --durations=10 --cov-append \
```

Comment:
did you try parallelizing this  in webserver tests?

Reply:
yes but I had the following issues:
- in webserver_isolated (e.g. the only valid candidate for now): some other library of pytest was giving me problems of compatibility so I did not look further.
- the other webserver ones I did not try too much since there is much to be changed regarding the problem of dependencies and session/module scoped fixtures that need to be adapted to run in parallel